src/server/graphql.spec.ts

```diff
@@ -18,104 +18,50 @@ describe("test graphql", () => {
-        type Query {
-          \\"\\"\\"
-          List all \`tasks\`
-          \\"\\"\\"
-          tasks (limit: Int, page: Int, orderBy: tasksOrderBy, where: tasksWhere): [tasks!]!
-          \\"\\"\\"
-          List all \`categories\`
-          \\"\\"\\"
-          categories (limit: Int, page: Int, orderBy: categoriesOrderBy, where: categoriesWhere): [categories!]!
+        "type Query {
+          task (id: ID!): Task
+          tasks (limit: Int, page: Int, orderBy: tasksOrderBy, where: tasksWhere): [Task!]!
+          category (id: ID!): Category
+          categories (limit: Int, page: Int, orderBy: categoriesOrderBy, where: categoriesWhere): [Category!]!
         }
 
-        \\"\\"\\"
-        Represents \`tasks\` entity.
-        \\"\\"\\"
-        type tasks {
-          \\"\\"\\"
-          Id
-          \\"\\"\\"
+        type Mutation {
+          createTask (input: CreateTaskInput!): CreateTaskPayload
+          updateTask (id: ID!, patch: UpdateTaskInput!): UpdateTaskPayload
+          deleteTask (id: ID!): DeleteTaskPayload
+          createCategory (input: CreateCategoryInput!): CreateCategoryPayload
+          updateCategory (id: ID!, patch: UpdateCategoryInput!): UpdateCategoryPayload
+          deleteCategory (id: ID!): DeleteCategoryPayload
+        }
+
+        type Task implements Node {
+          nodeId: ID!
           id: Int!
-          \\"\\"\\"
-          The Title
-          \\"\\"\\"
           title: String!
-          \\"\\"\\"
-          Is it completed?
-          \\"\\"\\"
           completed: Boolean!
-          \\"\\"\\"
-          Category
-          \\"\\"\\"
-          category: categories
+          category: Category
         }
 
-        \\"\\"\\"
-        Represents \`categories\` entity.
-        \\"\\"\\"
-        type categories {
-          \\"\\"\\"
-          Id
-          \\"\\"\\"
-          id: String!
-          \\"\\"\\"
-          Name
-          \\"\\"\\"
-          name: String!
-          \\"\\"\\"
-          List all \`tasks\` of \`categories\`
-          \\"\\"\\"
-          tasks (limit: Int, page: Int, orderBy: tasksOrderBy, where: tasksWhere): [tasks!]!
-        }
-
-        \\"\\"\\"
-        OrderBy options for \`tasks\`
-        \\"\\"\\"
         input tasksOrderBy {
           id: OrderBydirection
           title: OrderBydirection
           completed: OrderBydirection
           category: OrderBydirection
         }
 
-        \\"\\"\\"
-        OrderBy options for \`categories\`
-        \\"\\"\\"
-        input categoriesOrderBy {
-          id: OrderBydirection
-          name: OrderBydirection
-        }
-
-        \\"\\"\\"
-        Where options for \`tasks\`
-        \\"\\"\\"
         input tasksWhere {
-          id: tasksWhereid
-          title: tasksWheretitle
-          completed: tasksWherecompleted
-          category: tasksWherecategory
+          id: tasksWhere_id
+          title: tasksWhere_title
+          completed: tasksWhere_completed
```

Comment:
Do we really need a separate type for title etc... wouldn't it be netter to have one for string, one for number one for bool and work with them?

Reply:
Than I would start with less types - and if we need to expand, worry about that later. Agreed?

Reply:
Done, looks good and will not grows endless 💪